### PR TITLE
Connect Tourpage to MongoDB

### LINKED ISSUE #34: Connect "Tour"-Page to Database
- [x] #11  
- [x] #2 
- [ ] Connect "Tour"- Page to Database

### PR DESCRIPTION
**Pull request by lauravikanis**
Connect "Tour"- Page to Database -> show LocationNames
close #34

### CODE REVIEW

File: client/src/GlobalStyles.js [hunk 1]
```diff
@@ -43,8 +43,8 @@ h2 {
   font-size: 1.5rem;
   color: var(--primary-color);
 }
-li {
-  display:flex;
+li { 
+  list-style-position:inside;
```
Comment (ncbecker) [inline client/src/GlobalStyles.js:47]: Nice, didn't know this attribute 👍

File: client/src/pages/Choice.js [hunk 2]
```diff
@@ -25,13 +25,13 @@ const Home = () => {
         </Card>
       </Link>
       <Link to="tour">
-        <Card>
+        <Card hidden>
```
Comment (ncbecker) [inline client/src/pages/Choice.js:28]: If you don't want to render these components at all, you could also use conditional rendering

**Review by ncbecker — APPROVED** [review]
Lgtm, nice work 🙂👍  the deployment link didn't work for me

**Review by phlgr — COMMENTED** [review]
Nico made some valuable comments 👍

File: client/src/GlobalStyles.js (hunk 1 above)
Comment (lauravikanis) [reply client/src/GlobalStyles.js:47]: > Lgtm, nice work 🙂👍 the deployment link didn't work for me

Deployment is working now. forgot to add environments to heroku 🤦‍♀️

File: client/src/pages/Choice.js (hunk 2 above)
Comment (lauravikanis) [reply client/src/pages/Choice.js:28]: They are not supposed to be shown right now. so no other render method needed ✌️

**Pull request merged**